erc190/package.py

```diff
@@ -32,3 +31,16 @@ def __init__(self, lockfile):
         except jsonValidationError:
             raise ValidationError
         self.package_data = package_data
+
+    def __refr__(self):
```

Comment:
spelling: this should be `__repr__`

Reply:
I guess I was thinking `reference` instead of `representation`, oops. I just fixed it in the test too--that's why the CI failed.